third_party/BUILD.fmtlib

```diff
@@ -0,0 +1,11 @@
+cc_library(
+    name = "fmt",
```

Comment:
If you call this target `fmtlib` you can depend on it using just `@fmtlib` instead of `@fmtlib:fmt`.

Reply:
Nice!